Plots/Trajectories/NCL_traj_1.py

```diff
@@ -0,0 +1,86 @@
+"""
+NCL_traj_1.py
+===============
+This script illustrates the following concepts:
+   - Plotting a simple trajectory plot
+   - Plotting multiple trajectories in different colors
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/traj_1.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/traj_1_lg.png
+"""
+
+import xarray as xr
```

Comment:
We are using **Sphinx Python documentation generator** to populate our **ReadTheDocs** links with nice-looking styles, and I'd like to suggest some changes to the code (Generally comment lines) to make use of some features of Sphinx:

Before starting significant blocks within your code (e.g. package imports, data reading, or plotting lines), you can use following comment style:

```
###############################################################################
# Import packages:
```

to let Sphinx know to generate Jupyter notebook-like cells in our ReadTheDocs webpages. Example usages can be seen from previous functions.